internal/js/modules/k6/browser/common/page.go

```diff
@@ -823,7 +823,7 @@ func (p *Page) getOwnerFrame(apiCtx context.Context, h *ElementHandle) (cdp.Fram
 	}
 
 	action := dom.DescribeNode().WithObjectID(documentElement.remoteObject.ObjectID)
-	node, err := action.Do(cdp.WithExecutor(p.ctx, p.session))
+	node, err := action.Do(cdp.WithExecutor(p.ctx, h.session))
```

Comment:
Do you think it would be helpful to include [this commit](https://github.com/grafana/k6/pull/5071/commits/38a49a8ac468118ff77ed82f9a249e66636fe908)'s comment here? It might clear things up for anyone wondering why we're using the `ElementHandle` session instead of the `Page` session.

Reply:
Added in https://github.com/grafana/k6/pull/5086